src/interfaces/IEmailRecoveryManager.sol

```diff
@@ -33,9 +33,8 @@ interface IZkEmailRecovery {
         uint256 executeAfter; // the timestamp from which the recovery request can be executed
```

Comment:
consider using `uint48` here. would allow packing both `executeAfter` and `executeBefore` into one slot. since these are sloaded together